maven-central-sync/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+
+LOCAL_REPO_DIR=$1
+
+CMD="mvn org.sonatype.plugins:nexus-staging-maven-plugin:1.6.7:"
+DEFAULT_OPTS="-DnexusUrl=https://s01.oss.sonatype.org/ -DserverId=ossrh"
+PROFILE_OPT="-DstagingProfileId=13c1877339a4cf"
```

Comment:
Can you add a comment about this staging ID, where does it come from?

Reply:
I will add a section to the readme